### DISCUSSION TITLE
New Action: Remove from album

### DISCUSSION THREAD

**Discussion by Fix-MX** [xob0t/Google-Photos-Toolkit#23] [Ideas]
I started adding all space consuming photos to a new album. It worked perfectly and did what I expected. 

Now I was going to trash all space consuming photos, but I realized that Collages were there and I want to keep them. So I used the toolkit again to add collages (by name filtering) to a new album called collages. After that, collages are in both SpaceConsuming and Collages albums.

Now I want to remove collages from SpaceConsuming album so I can trash the entire SpaceConsuming album without trashing collages that are still in this album, but this action is not implemented, right? 

It would be nice to have this action. I don't know if it's feasible or not using the API, but it is available from the three dot menu when you open a photo in an album.

By the way, great job!. Google should have many of this actions implemented directly on their application.

**Comment by xob0t** [comment]
You're right, GPTK does not have an action to remove items from an album.
But it is also not needed in your case. In the Albums source, select your SpaceConsuming album in the `Select` filter, then select Collages album in `Exclude` filter.
Now you're good to run the `move to trash` action, that will only apply to items that are in SpaceConsuming, but not in Collages.
If you want to be extra sure before moving items to trash, you can first create a new album using that filter, then move that album to trash after you inspect its contents and its all good.
>By the way, great job!. Google should have many of this actions implemented directly on their application.

Thank you for the kind words :D

**Reply by Wins911** [reply]
Hi xob0t. Thanks for the script. Is any chance you will add this function - move out from one album to another, or just delete photos only from speciphic album? This will alloud to arrange the photos in better way.

**Reply by xob0t** [reply]
I've created an issue to track it - #65
